interface/main/calendar/modules/PostCalendar/pnincludes/overlib_mini.js

```diff
@@ -241,514 +309,718 @@ var fnRef;
-}else if(ns6){
-if(frameid.document.getElementById('overDiv')==null)return false;
-}
-return true;
-}
-function layerWrite(txt){
-txt +="\n";
-if(ns4){
-var lyr=oframe.document.overDiv.document
-lyr.write(txt)
-lyr.close()
-}else if(ie4){
-oframe.document.all["overDiv"].innerHTML=txt
-}else if(ns6){
-range=oframe.document.createRange();
-range.setStartBefore(over);
-domfrag=range.createContextualFragment(txt);
-while(over.hasChildNodes()){
-over.removeChild(over.lastChild);
-}
-over.appendChild(domfrag);
-}
-}
-function showObject(obj){
-if(ns4)obj.visibility="show";
-else if(ie4)obj.visibility="visible";
-else if(ns6)obj.style.visibility="visible";
-}
-function hideObject(obj){
-if(ns4)obj.visibility="hide";
-else if(ie4)obj.visibility="hidden";
-else if(ns6)obj.style.visibility="hidden";
-if(otimerid > 0)clearTimeout(otimerid);
-if(odelayid > 0)clearTimeout(odelayid);
-otimerid=0;
-odelayid=0;
-self.status="";
-}
-function repositionTo(obj,xL,yL){
-if((ns4)||(ie4)){
-obj.left=(ie4 ? xL + 'px' : xL);
-obj.top=(ie4 ? yL + 'px' : yL);
-}else if(ns6){
-obj.style.left=xL + "px";
-obj.style.top=yL+ "px";
-}
-}
-function getFrameRef(thisFrame, ofrm){
-var retVal='';
-for(var i=0;i<thisFrame.length;i++){
-if(thisFrame[i].length > 0){
-retVal=getFrameRef(thisFrame[i],ofrm);
-if(retVal=='')continue;
-}else if(thisFrame[i] !=ofrm)continue;
-retVal='['+i+']' + retVal;
-break;
-}
-return retVal;
-}
-function opt_FRAME(frm){
-oframe=compatibleframe(frm)? frm : ol_frame;
-if(oframe !=ol_frame){
-var tFrm=getFrameRef(top.frames, oframe);
-var sFrm=getFrameRef(top.frames, ol_frame);
-if(sFrm.length==tFrm.length){
-l=tFrm.lastIndexOf('[');
-if(l){
-while(sFrm.substring(0,l)!=tFrm.substring(0,l))l=tFrm.lastIndexOf('[',l-1);
-tFrm=tFrm.substr(l);
-sFrm=sFrm.substr(l);
-}
-}
-var cnt=0, p='', str=tFrm;
-while((k=str.lastIndexOf('['))!=-1){
-cnt++;
-str=str.substring(0,k);
-}
-for(var i=0;i<cnt;i++)p=p + 'parent.';
-fnRef=p + 'frames' + sFrm + '.';
-}
-if((ns4)||(ie4 ||(ns6))){
-if(ns4)over=oframe.document.overDiv;
-if(ie4)over=oframe.overDiv.style;
-if(ns6)over=oframe.document.getElementById("overDiv");
-}
-return 0;
-}
-function opt_FUNCTION(callme){
-otext=(callme ? callme():(ofunction ? ofunction(): 'No Function'));
-return 0;
+if (ie4) {
+    var docRoot='document.body';
 }
+var ie5=false;
+if (ns4) {
+    var oW=window.innerWidth;
+    var oH=window.innerHeight;
+    window.onresize=function () {
+        if (oW!=window.innerWidth||oH!=window.innerHeight) {
+            location.reload();
+        }}
+}
+if (ie4) {
+    if ((navigator.userAgent.indexOf('MSIE 5')> 0)||(navigator.userAgent.indexOf('MSIE 6')> 0)) {
+        if (document.compatMode && document.compatMode=='CSS1Compat') {
+            docRoot='document.documentElement';
+        }
+        ie5=true;
+    }
+    if (ns6) {
+        ns6=false;
+    }
+}
+if ((ns4)||(ie4)||(ns6)) {
+    document.onmousemove=mouseMove
+    if (ns4) {
+        document.captureEvents(Event.MOUSEMOVE)
+    } } else {
```

Comment:
Thought I found an error here with 2 brackets, but I didn't. Like trying to beat a computer in chess; not possible :)